src/selectors/per-thread/markers.js

```diff
@@ -107,14 +107,15 @@ export function getMarkerSelectorsPerThread(threadSelectors: *) {
     Marker[]
   > = createSelector(getCommittedRangeFilteredMarkers, (markers): Marker[] =>
     markers.filter(
-      tm =>
-        tm.name !== 'GCMajor' &&
-        tm.name !== 'BHR-detected hang' &&
-        tm.name !== 'LongTask' &&
-        tm.name !== 'LongIdleTask' &&
-        tm.name !== 'Jank' &&
-        !MarkerData.isNetworkMarker(tm) &&
-        !MarkerData.isNavigationMarker(tm)
+      marker =>
+        marker.name !== 'GCMajor' &&
+        marker.name !== 'BHR-detected hang' &&
+        marker.name !== 'LongTask' &&
+        marker.name !== 'LongIdleTask' &&
+        marker.name !== 'Jank' &&
+        !MarkerData.isNetworkMarker(marker) &&
+        !MarkerData.isDiskIoMarker(marker) &&
+        !MarkerData.isNavigationMarker(marker)
```

Comment:
ah, eventually we should have a `shouldDisplayMarkerInHeader(marker)` boolean utility function :) (optional for this PR)